README.md

```diff
@@ -82,7 +93,14 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+&lt;noun&gt; ::= "dog" | "cat"  
+&lt;verb&gt; ::= "ate" | "ran"  
+&lt;adjective&gt; ::= "" | "lazy" | "smelly"  
+&lt;adverb&gt; ::= "" | "slowly" | "noisily"  
+
+&lt;adjlist&gt; ::= ""  
+&lt;adjlist&gt; ::= &lt;adjlist&gt; | &lt;adjective&gt;  
+&lt;sentence&gt; ::= "the" &lt;adjlist&gt; &lt;noun&gt; &lt;verb&gt; &lt;adverb&gt;  
```

Comment:
Any idea why these all ended up &lt; and &gt;?

# 6

So close... Line 102 should have been

    <adjlist> ::= <adjlist> <adjective>